crates/lair_keystore/src/bin/lair-keystore-bin/main.rs

```diff
@@ -50,13 +33,14 @@ pub(crate) async fn read_interactive_passphrase(
     .await
     .map_err(one_err::OneErr::new)??;
 
-    vec_to_locked(pass_tmp)
+    Ok(sodoken::LockedArray::from(pass_tmp))
```

Comment:
vec_to_locked used to also zero the pass_tmp buffer.

Reply:
I propose fixing this in `sodoken`, we move the vector here so we can't clear the source here I don't think -> https://github.com/holochain/sodoken/pull/29

Reply:
Suggested using zeroize crate in sodoken pr, but yes, make sense to solve it there :+1:

Reply:
Done by https://github.com/holochain/sodoken/pull/29 and bumped here https://github.com/holochain/lair/pull/143/commits/df30ed8a7f880f9e3f7df90266303b8d8d60eca4